base/essentials.jl

```diff
@@ -17,12 +17,21 @@ abstract type AbstractSet{T} end
 """
     AbstractDict{K, V}
 
-Supertype for dictionary-like types with keys of type `K` and values of type `V`.
-[`Dict`](@ref), [`IdDict`](@ref) and other types are subtypes of this.
-An `AbstractDict{K, V}` should be an iterator of `Pair{K, V}`.
+Supertype for unordered (and ordered) dictionary-like types with keys of type `K`
+and values of type `V`. [`Dict`](@ref), [`IdDict`](@ref) and other types
+are subtypes of this. An `AbstractDict{K, V}` should be an iterator of `Pair{K, V}`.
 """
 abstract type AbstractDict{K,V} end
 
+"""
+    AbstractOrderedDict{K, V}
+
+Supertype for ordered dictionary-like types with keys of type `K` and values of
+type `V`. [`Dict`](@ref), [`IdDict`](@ref) and other types are subtypes of this.
```

Comment:
They're not?

Reply:
Ok, sorry you're right, a bit premature... They're only ordered on my other PR (I thought it was on the finish-line, but it maight not make it). You like be to skip that part to merge, or wait for my other PR to be merged first?